Training5/src/task1/src/reducers/todo.js

```diff
@@ -0,0 +1,122 @@
+const initialState = {
+  Todos: [],
+  isOpenForm: false,
+  editTodo: {},
```

Comment:
@phuongluongZigvy  as I said above, you don't need store editTodo, only editingTodoId

Reply:
I don't see searchValue in the state